static/app/components/modals/inviteMembersModal/inviteMembersFooter.tsx

```diff
@@ -0,0 +1,54 @@
+import {Fragment, useContext} from 'react';
+import styled from '@emotion/styled';
+
+import ButtonBar from 'sentry/components/buttonBar';
+import InviteButton from 'sentry/components/modals/inviteMembersModal/inviteButton';
+import {InviteMembersContext} from 'sentry/components/modals/inviteMembersModal/inviteMembersContext';
+import InviteStatusMessage from 'sentry/components/modals/inviteMembersModal/inviteStatusMessage';
+import {space} from 'sentry/styles/space';
+
+interface Props {
+  canSend: boolean;
+}
+
+export default function InviteMembersFooter({canSend}: Props) {
+  const {complete, inviteStatus, invites, sendInvites, sendingInvites, willInvite} =
+    useContext(InviteMembersContext);
+  const isValidInvites = invites.length > 0;
+
+  return (
+    <FooterContent>
+      <div>
+        {/* TODO(mia): remove these props and use InviteMemberContext once old modal is removed */}
```

Comment:
if you wanted.. you could move the context to wrap both components - then refactor these as well. (if you wanted to do that, it'd probably be better as a separate PR tho)